pages/components/Login.tsx

```diff
@@ -0,0 +1,71 @@
+import axios from 'axios';
+import router, { useRouter } from 'next/router';
+import { useState } from 'react';
+
+
+const Login = () => {
+
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleSubmit = async (e: React.FormEvent) => {
+        e.preventDefault();
+
+        try{
+            const response = await axios.post('/api/auth/Login', { email, password });
+            localStorage.setItem('token', response.data);
+            router.push('/components/Todolist')
```

Comment:
why naming routes with `/components`, looks wrong

Reply:
without the /components it doesn't work sir

Reply:
restructure your folder

Reply:
I am using Pages router sir, and inside my pages file I have the api(for auth) and the components for the (client side tsx).. how can i restructure it sir?